### ISSUE TITLE
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'guardvillagers'!

### ISSUE THREAD

**Issue by joex92** [mrsterner/GuardVillagers#48]
It just crashes on dedicated server Fabric 1.19.2

<details>
<summary>latest.log</summary>

```console
[21:31:40] [main/INFO]: Loading Minecraft 1.19.2 with Fabric Loader 0.14.11
[21:31:41] [ForkJoinPool-1-worker-10/WARN]: Mod globaldataandresourcepacks uses the version 1.14.2_fabric which isn't compatible with Loader's extended semantic version format (Could not parse version number component '2_fabric'!), SemVer is recommended for reliably evaluating dependencies and prioritizing newer version
[21:31:41] [ForkJoinPool-1-worker-12/WARN]: Mod inventorytabs uses the version 0.9.beta-1.19.x which isn't compatible with Loader's extended semantic version format (Could not parse version number component 'beta'!), SemVer is recommended for reliably evaluating dependencies and prioritizing newer version
[21:31:41] [main/INFO]: Loading 102 mods:
	- at_favre_lib_bcrypt 0.9.0 via easyauth
	- at_favre_lib_bytes 1.5.0 via easyauth
	- audioplayer 1.19.2-1.4.5
	- carpet 1.4.84
	- carpet-extra 1.4.84
	- carpet-fixes 1.19-1.12.2
	- chunky 1.3.50
	- cloth-basic-math 0.6.1 via cloth-config
	- cloth-config 8.2.88
	- com_electronwill_night-config_core 3.6.5 via globaldataandresourcepacks
	- com_electronwill_night-config_toml 3.6.5 via globaldataandresourcepacks
	- com_fasterxml_jackson_core_jackson-annotations 2.12.6 via discarpet
	- com_fasterxml_jackson_core_jackson-core 2.12.6 via discarpet
	- com_fasterxml_jackson_core_jackson-databind 2.12.6 via discarpet
	- com_github_llamalad7_mixinextras 0.0.6 via carpet-fixes
	- com_neovisionaries_nv-websocket-client 2.14 via discarpet
	- com_squareup_okhttp3_logging-interceptor 4.9.3 via discarpet
	- com_squareup_okhttp3_okhttp 4.9.3 via discarpet
	- com_squareup_okio_okio 2.8.0 via discarpet
	- com_typesafe_config 1.4.2 via thorium
	- conditional-mixin 0.3.1 via carpet-fixes
	- coordfinder 1.19.2-1.0.1
	- ctov 3.1.4
	- de_mkammerer_argon2-jvm 2.11 via easyauth
	- de_mkammerer_argon2-jvm-nolibs 2.11 via easyauth
	- diamondeconomy 1.5.4
	- discarpet 1.4.7
	- easyauth 3.0.0-8
	- fabric-api 0.69.0+1.19.2
	- fabric-api-base 0.4.13+0d0f210290 via fabric-api
	- fabric-api-lookup-api-v1 1.6.12+93d8cb8290 via fabric-api
	- fabric-biome-api-v1 9.0.20+0d0f210290 via fabric-api
	- fabric-block-api-v1 1.0.1+12bfe4ea90 via fabric-api
	- fabric-command-api-v1 1.2.13+f71b366f90 via fabric-api
	- fabric-command-api-v2 2.1.9+93d8cb8290 via fabric-api
	- fabric-commands-v0 0.2.30+df3654b390 via fabric-api
	- fabric-containers-v0 0.1.38+df3654b390 via fabric-api
	- fabric-content-registries-v0 3.4.4+6064db2a90 via fabric-api
	- fabric-convention-tags-v1 1.2.0+27265b3490 via fabric-api
	- fabric-crash-report-info-v1 0.2.7+aeb40ebe90 via fabric-api
	- fabric-data-generation-api-v1 5.3.3+ad31814d90 via fabric-api
	- fabric-dimensions-v1 2.1.33+0d0f210290 via fabric-api
	- fabric-entity-events-v1 1.5.2+9244241690 via fabric-api
	- fabric-events-interaction-v0 0.4.31+6eb9a2a190 via fabric-api
	- fabric-events-lifecycle-v0 0.2.31+df3654b390 via fabric-api
	- fabric-game-rule-api-v1 1.0.23+c6af733c90 via fabric-api
	- fabric-item-api-v1 1.6.1+fa140d5990 via fabric-api
	- fabric-item-groups-v0 0.3.36+9244241690 via fabric-api
	- fabric-lifecycle-events-v1 2.2.2+9244241690 via fabric-api
	- fabric-loot-api-v2 1.1.10+83a8659290 via fabric-api
	- fabric-loot-tables-v1 1.1.13+9e7660c690 via fabric-api
	- fabric-message-api-v1 5.0.5+93d8cb8290 via fabric-api
	- fabric-mining-level-api-v1 2.1.21+33fbc73890 via fabric-api
	- fabric-networking-api-v1 1.2.8+3b1d070f90 via fabric-api
	- fabric-networking-v0 0.3.25+df3654b390 via fabric-api
	- fabric-object-builder-api-v1 4.1.3+06e0e25190 via fabric-api
	- fabric-particles-v1 1.0.12+79adfe0a90 via fabric-api
	- fabric-permissions-api-v0 0.1-SNAPSHOT via easyauth
	- fabric-registry-sync-v0 0.9.29+9244241690 via fabric-api
	- fabric-rendering-data-attachment-v1 0.3.17+0d0f210290 via fabric-api
	- fabric-rendering-fluids-v1 3.0.9+c6af733c90 via fabric-api
	- fabric-resource-conditions-api-v1 2.1.1+aae9039d90 via fabric-api
	- fabric-resource-loader-v0 0.8.2+574a764890 via fabric-api
	- fabric-screen-handler-api-v1 1.3.4+1cc24b1b90 via fabric-api
	- fabric-transfer-api-v1 2.1.3+9d081d6490 via fabric-api
	- fabric-transitive-access-wideners-v1 1.3.2+42d99c3290 via fabric-api
	- fabricloader 0.14.11
	- flan 1.19.2-1.8.2
	- globaldataandresourcepacks 1.14.2_fabric
	- guardvillagers 1.19.2-1.0.16
	- io_leangen_geantyref_geantyref 1.3.13 via thorium
	- io_vavr_vavr 0.10.4 via discarpet
	- itemswapper 0.3.2-mc1.19.2
	- java 17
	- lazydfu 0.1.3
	- lithium 0.10.4
	- minecraft 1.19.2
	- mysql_mysql-connector-java 8.0.29 via easyauth
	- net_java_dev_jna_jna 5.12.1 via easyauth
	- notenoughcrashes 4.2.0+1.19.2
	- org_iq80_leveldb_leveldb 0.12 via easyauth
	- org_iq80_leveldb_leveldb-api 0.12 via easyauth
	- org_javacord_javacord-api 3.5.0 via discarpet
	- org_javacord_javacord-core 3.5.0 via discarpet
	- org_jetbrains_kotlin_kotlin-stdlib 1.4.10 via discarpet
	- org_mongodb_bson 4.7.0 via easyauth
	- org_mongodb_mongodb-driver-core 4.7.0 via easyauth
	- org_mongodb_mongodb-driver-sync 4.7.0 via easyauth
	- org_spongepowered_configurate-core 4.1.2 via thorium
	- org_spongepowered_configurate-hocon 4.1.2 via thorium
	- org_xerial_sqlite-jdbc 3.36.0.3 via diamondeconomy
	- packet_tweaker 0.3.0+1.18.2 via server_translations_api
	- phosphor 0.8.1
	- placeholder-api 2.0.0-beta.7+1.19 via easyauth
	- rtree-3i-lite-fabric 0.3.0 via flan
	- server_translations_api 1.4.16+1.19 via easyauth
	- status 1.19.2-1.0.0
	- stoneholm 1.4.4
	- thorium 1.3.2
	- viewdistancefix 1.19.2-1.0.0
	- voicechat 1.19.2-2.3.22
	- worldedit 7.2.12+6240-87f4ae1
[21:31:41] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/C:/Users/joex92/odrive/GoogleDrive%5bFIEC%5d/Minecraft/Backups/JoeX/2022-12-12_Fabric-1.19.2/libraries/net/fabricmc/sponge-mixin/0.11.4+mixin.0.8.5/sponge-mixin-0.11.4+mixin.0.8.5.jar Service=Knot/Fabric Env=SERVER
[21:31:41] [main/INFO]: Compatibility level set to JAVA_16
[21:31:41] [main/INFO]: Compatibility level set to JAVA_17
[21:31:42] [main/INFO]: Loaded configuration file for Lithium: 106 options available, 0 override(s) found
[21:31:44] [main/WARN]: [carpet] detected, disabling UPDATE mixin com.sk89q.worldedit.fabric.mixin.MixinLevelChunkSetBlockHook
[21:31:46] [main/INFO]: Downloading deobfuscation mappings: 1.19.2+build.28 for the first launch
[21:31:51] [main/WARN]: Method overwrite conflict for getLanguage in worldedit-fabric.mixins.json:MixinServerPlayer from mod worldedit, previously written by carpet.mixins.ServerPlayer_scarpetEventMixin. Skipping method.
[21:31:52] [main/WARN]: Method overwrite conflict for method_5800 in carpet-fixes.mixins.json:itemFixes.ItemEntity_lightningKillsDropsMixin from mod carpet-fixes, previously written by carpet.mixins.ItemEntityMixin. Skipping method.
[21:31:57] [main/INFO]: Building unoptimized datafixer
[21:32:04] [main/INFO]: Bot 750064706408153089 sucessfully logged in
[21:32:04] [main/INFO]: Discarpet loaded
[21:32:04] [main/INFO]: [EasyAuth] EasyAuth mod by samo_lego, NikitaCartes.
[21:32:05] [main/INFO]: Compatibility version 1
[21:32:05] [main/INFO]: Compatibility version 15
[21:32:05] [main/INFO]: Loading plugins
[21:32:05] [main/INFO]: Loaded 1 plugin(s)
[21:32:05] [main/INFO]: Initializing plugins
[21:32:05] [main/INFO]: Initialized 1 plugin(s)
[21:32:05] [main/INFO]: Registering events for audioplayer
[21:32:06] [main/INFO]: Got request to register class com.sk89q.worldedit.fabric.FabricPlatform with WorldEdit [com.sk89q.worldedit.extension.platform.PlatformManager@6f8ac388]
[21:32:06] [main/INFO]: WorldEdit for Fabric (version 7.2.12+6240-87f4ae1) is loaded
[21:32:06] [main/ERROR]: Failed to start the minecraft server
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'guardvillagers'!
	at Not Enough Crashes deobfuscated stack trace.(1.19.2+build.28) ~[?:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51) ~[fabric-loader-0.14.11.jar:?]
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33) ~[fabric-loader-0.14.11.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49) ~[fabric-loader-0.14.11.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35) ~[fabric-loader-0.14.11.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startServer(Hooks.java:62) ~[fabric-loader-0.14.11.jar:?]
	at net.minecraft.server.Main.main(Main:101) [server-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461) [fabric-loader-0.14.11.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.14.11.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) [fabric-loader-0.14.11.jar:?]
	at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:69) [fabric-loader-0.14.11.jar:?]
Caused by: java.lang.VerifyError: Bad access to protected data in invokevirtual
Exception Details:
  Location:
    dev/mrsterner/guardvillagers/common/entity/GuardEntity.lambda$initGoals$3(Lnet/minecraft/class_1309;)Z @17: invokevirtual
  Reason:
    Type 'net/minecraft/class_1309' (current frame, stack[1]) is not assignable to 'dev/mrsterner/guardvillagers/common/entity/GuardEntity'
  Current Frame:
    bci: @17
    flags: { }
    locals: { 'net/minecraft/class_1309' }
    stack: { 'java/util/List', 'net/minecraft/class_1309' }
  Bytecode:
    0000000: 2ac1 06a6 9900 1cb2 0314 b403 18b4 06aa
    0000010: 2ab6 06ae b906 af02 009a 0007 04a7 0004
    0000020: 03ac                                   
  Stackmap Table:
    same_frame(@32)
    same_locals_1_stack_item_frame(@33,Integer)

	at dev.mrsterner.guardvillagers.GuardVillagers.<clinit>(GuardVillagers.java:44) ~[guard-villagers-fabric-1.19.2-1.0.16.jar:?]
	at java.lang.Class.forName0(Native Method) ~[?:?]
	at java.lang.Class.forName(Unknown Source) ~[?:?]
	at net.fabricmc.loader.impl.util.DefaultLanguageAdapter.create(DefaultLanguageAdapter.java:50) ~[fabric-loader-0.14.11.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointStorage$NewEntry.getOrCreate(EntrypointStorage.java:117) ~[fabric-loader-0.14.11.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:53) ~[fabric-loader-0.14.11.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-loader-0.14.11.jar:?]
	... 7 more

```
</details>

**Comment by YumatanGames** [comment]
I'm also having the same issue.
[message.txt](https://github.com/mrsterner/GuardVillagers/files/10215265/message.txt)

**Comment by mrsterner** [comment]
fixed in 1.0.18
